docs/api/Badge/Badge.md

```diff
@@ -11,7 +11,7 @@ Props
 | accent | bool | false | If true, the badge will use the accent badge colors. |
 | badgeClassName | string |  | The CSS class name of the badge element. |
 | <span style="color: #31a148">badgeContent *</span> | node |  | This is the content rendered within the badge. |
-| children | node |  | The badge will be added relativelty to this node. |
+| <span style="color: #31a148">children *</span> | node |  | The badge will be added relativelty to this node. |
```

Comment:
Typo in `relatively`, but it should probably be `relative`, and even then the sentence is awkward.

How about simply "The node that the badge will be applied to."